patches/server/0034-IntelliJ-Remove-Streams.patch

```diff
@@ -0,0 +1,603 @@
+     // Paper start
+     @Override
+     public List<net.md_5.bungee.api.chat.BaseComponent[]> getLoreComponents() {
+-        return this.lore == null ? null : new ArrayList<>(this.lore.stream().map(entry ->
+-            net.md_5.bungee.chat.ComponentSerializer.parse(IChatBaseComponent.ChatSerializer.componentToJson(entry)
+-        )).collect(java.util.stream.Collectors.toList()));
++        if (this.lore == null) {
++            return null;
++        } else {
++            List<BaseComponent[]> list = new ArrayList<>();
++            for (IChatBaseComponent entry : this.lore) {
++                BaseComponent[] parse = ComponentSerializer.parse(IChatBaseComponent.ChatSerializer.componentToJson(entry)
++                );
++                list.add(parse);
++            }
++            return new ArrayList<>(list);
++        }
+     }
+     // Paper end
+     @Override
+diff --git a/src/test/java/com/destroystokyo/paper/entity/ai/VanillaMobGoalTest.java b/src/test/java/com/destroystokyo/paper/entity/ai/VanillaMobGoalTest.java
+index 232c382b586b0812c9f7161565c0d382177adf7d..ac1c7160d45f88008bba68184045e79b5e647fa9 100644
+--- a/src/test/java/com/destroystokyo/paper/entity/ai/VanillaMobGoalTest.java
++++ b/src/test/java/com/destroystokyo/paper/entity/ai/VanillaMobGoalTest.java
+@@ -43,11 +43,15 @@ public class VanillaMobGoalTest {
+             classes = scanResult.getSubclasses("net.minecraft.server.PathfinderGoal").loadClasses();
+         }
+ 
+-        List<GoalKey<?>> vanillaNames = classes.stream()
+-            .filter(VanillaMobGoalTest::hasNoEnclosingClass)
+-            .filter(clazz -> !Modifier.isAbstract(clazz.getModifiers()))
+-            .map(goalClass -> MobGoalHelper.getKey((Class<? extends PathfinderGoal>) goalClass))
+-            .collect(Collectors.toList());
++        List<GoalKey<?>> vanillaNames = new ArrayList<>();
++        for (Class<?> clazz : classes) {
++            if (hasNoEnclosingClass(clazz)) {
++                if (!Modifier.isAbstract(clazz.getModifiers())) {
++                    GoalKey<Mob> mobGoalKey = MobGoalHelper.getKey((Class<? extends PathfinderGoal>) clazz);
++                    vanillaNames.add(mobGoalKey);
++                }
++            }
++        }
+ 
+         List<GoalKey<?>> missingFromAPI = new ArrayList<>(vanillaNames);
+         missingFromAPI.removeAll(keys);
+@@ -66,13 +70,17 @@ public class VanillaMobGoalTest {
+         }
+         if (missingFromVanilla.size() != 0) {
+             System.out.println("Missing from vanilla: ");
+-            missingFromVanilla.forEach(System.out::println);
++            for (GoalKey<?> goalKey : missingFromVanilla) {
++                System.out.println(goalKey);
++            }
+             shouldFail = true;
+         }
+ 
+         if (deprecated.size() != 0) {
+             System.out.println("Deprecated (might want to remove them at some point): ");
+-            deprecated.forEach(System.out::println);
```

Comment:
This is literally a test. Miniscule performance gains do not matter. Also not a stream.